zarr.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: idr
+title: Zarr
+redirect_from:
+  - zarr/
+---
+
+        <!-- begin marketing header -->
+        <header class="marketing-hero">
+            <div class="row homepage text-center">
+
+                <div class="medium-12 columns">
+
+                    <h1 class="hero-main-header">Cloud access</h1>
+                    <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                    The imaging data for some studies published in the
+                    Image Data Resource (IDR) is also available in a converted
+                    form on S3.
+                    </p>
+                </div>
+            </div>
+        </header>
+
+        <!-- begin Example section -->
+
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-10 small-centered medium-10 medium-centered columns">
+                <div class="row horizontal">
+                    <div>
+                        
+                        TODO: include zarr.md
```

Comment:
This might introduce a dependency of this PR on #107, where the content of `zarr.md` is introduced. @manics is there a way to introduce some mark-up that would create an anchor?

Reply:
Is this PR required for the upcoming conferences? If not can we deal with the addition of the anchor later?

Reply:
To the best of my knowledge, this PR is not a requirement for the upcoming conference. I am completely fine with having #107 merged and deployed first.
The anchor and the cross-link could be added as part of this PR assuming this is possible using `mardownify`

Reply:
> this PR is not a requirement for the upcoming conference.

I'm currently not linking to it.